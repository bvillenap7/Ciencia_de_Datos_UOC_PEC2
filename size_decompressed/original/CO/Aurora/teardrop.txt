Noticed a bright light Similar to a plane when it’s facing you, and then a second later it dropped really fast and disappeared.;2019-12-18 00:28:00;1 minute;Noticed a bright light Similar to a plane when it’s facing you, and then a second later it dropped really fast and disappeared.;39.6695444272445;-104.803991486068
