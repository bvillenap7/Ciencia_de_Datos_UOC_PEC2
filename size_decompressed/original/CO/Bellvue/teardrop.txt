Teardrop shape that created a tail flying low and parallel to the gound.;2002-09-06 20:30:00;2 min.;Teardrop shape that created a tail flying low and parallel to the gound. A teardrop shape flying parallel to the horizon, flying close enough that we thought it might have been a helicopter at first. Bright, solid, teardrop that started emitting a tail as it moved in and out of samll clouds for a couple of seconds. Red and blue in the tail. The tail got longer and samll bursts of light appeared to pulse from the tail. It then entered another small cloud and never reappeared. We are husband and wife, never seen anything like this before. Husband is a ((deleted--senior academician)). Wife works in a technical/engineering field.;40.6265;-105.261
