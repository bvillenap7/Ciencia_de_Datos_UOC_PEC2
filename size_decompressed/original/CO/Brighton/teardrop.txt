My husband and I saw a bright white light with a short tail glidding fast accross the sky from east to west.;2003-10-04 18:10:00;45-60 seconds;My husband and I saw a bright white light with a short tail glidding fast accross the sky from east to west. My husband and I were sitting outside this evening around 18:10 Mountain time taking in the cool autumn air when my husband looked up and seen a bright white light with  a short tail.  I looked up as well and saw the same thing.  It was teardrop shaped and glidding along pretty fast.  My father is a pilot and I learned enough to know from him that the object was not a military jet, airliner, hellicopter, or small airplane. There was no sound what so ever from this object.  We watched it for a good 45 seconds to about a minute.  We know that it was not a ameter cause it was moving too slow to be a meter. The object was heading west from the east.;39.948859685864;-104.816636125655
