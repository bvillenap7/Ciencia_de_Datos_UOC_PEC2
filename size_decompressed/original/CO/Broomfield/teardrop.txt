We were driving home from a coffee shop in Broomfield and we thought we heard a police siren so we pulled over but we didn't see any em;1998-04-21 22:30:00;2 minutes;We were driving home from a coffee shop in Broomfield and we thought we heard a police siren so we pulled over but we didn't see any emergency vehicles.  My friend was in the back seat looking out the back window and she shouted to us to look.  There were 3 tear dropped shaped objects in a triangle formation, the radio station went out and we just heard static for 2 minutes.  Then the objects just dissapeared. The radio came back on. there were 3 tear dropped shaped objects, they were silver but not glowing, they were metallic and shinning.  Hoovered and then dissappeared after a few minutes.;39.9141244186047;-105.074105813953
Horizontal gray teardrop shaped craft arced across neighborhood  just above houses;2017-02-25 13:15:00;2 minutes;horizontal gray teardrop shaped craft arced across neighborhood just above houses. I was on my way home from the grocery store at about 1:15pm on 25 February, 2017, heading south on Kohl St. in Broomfield, CO.  As I stopped at the intersection of W. 11th, looking southward, I saw a strange object moving above the roof tops near the intersection of Kohl St. and W. 10th Avenue, a block away. The object was a solid and somewhat irregularly shaped teardrop that was horizontal and heading at an even level and speed towards the east.  It was a gunmetal gray color.  The plumper part of the object was to the east and the sharper point to the west.  It was smooth, but on the east end, the shape had a slightly squared edge on the top rim.  The bottom of it seemed flat.  I wondered if it were a drone, but it had no protruding parts, such as propellers or wings.  I am estimating that this object was about two feet long and two and a half feet at the widest point. There was no traffic, so I turned east on W. 11th and kept looking at it as it arced northeast above the houses on the next street.  I turned down that street, but it flew beyond the street and disappeared from view. The wind was pretty strong from the northwest at that time.  If it had been debris, I believe it would have tumbled and pirouetted in the sky, taking on irregular shapes.  Instead, it remained in the same shape, and it moved in a completely even trajectory, at a steady altitude, and at a steady pace.  I do not believe this was debris caught in the wind.  I believe it was a craft moving at a steady velocity, with purpose, towards the east/northeast. If it made any sound, I did not hear it because I was in the car and the windows were rolled up completely.;39.9141244186047;-105.074105813953
