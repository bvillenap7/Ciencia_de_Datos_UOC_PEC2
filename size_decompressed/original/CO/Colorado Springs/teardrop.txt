Craft near AFB followed aircraft then proceeded south at a constant rate through changes in direction and elevation;2006-06-02 10:40:00;10-12 seconds;Craft near AFB followed aircraft then proceeded south at a constant rate through changes in direction and elevation he shape was not clear since the color of the craft matched the sky haze.  The only reason the craft was so visible was the reflection it gave off but the reflection did not show the exact outline.  However the reflection(s) allowed use to visually follow the craft while my friend and I kept asking each other, Have you ever seen an aircraft do that?, over and over.  Al was over Saturday evening and I saw my first UFO.  Craziest thing I've ever seen.  I still don't believe it was an Alien thing but it wasn't any typical aircraft.  The speed and its maneuvering looked impossible.  It chased two planes over the airport going two different directions, dodged another and then dove close to the ground and proceeded south all in 10-12 seconds.  It was at dusk so the sun was going down but shinning on the vehicle.  It was white and no lights of any kind.  It had to be a pearl white or like that reflective 3M white tape or sign material.  It was not shiny metallic but definitely white but a halogen white.  It did not have a vertical stab, rudder and it was a little hazy so there was no way to make out any detail.  There has been a lot of activity at the Ft. Carson and Peterson since this weekend.  My guess is that it is some sort of new flight vehicle and they were testing it.  It was likely also very stealthy so I assume it was likely not picked up on radar, especially considering what it did and the speed involved.  It was directly east when I saw it and farther out than an approach for the airport.  It came around and come up behind the first plane and veered off coming around to approaching the second plane coming in at a right angle to the first plane.  That was an 'S' shape that covered more than 12 miles.  Then it went south and dodged a plane on approach while in a steep dive.  After it came close to the ground it leveled off and proceeded south.  It did this all within 8 seconds, no longer.  It did not seem to lean into turns as an aircraft would and looked like it stayed level.  Neither Al or I could make out its shape except it did not have a rudder.  The only plane that I know that could do this would be an F-22 but there was no sound.  Something maneuvering that fast should have made some pretty noticeable throttling jet engine sounds.  If the F-22 has noise suppression maybe, flying an experimental pattern and painted pearl or reflective white.  My guess and feel after just watching the Thunderbirds at graduation is it was traveling just under mach one.  Also it did not change speed either in the hard maneuvers or when it dove.  It just looked really odd.  Neither Al or I took our eyes off of it until we could not see it any longer and just started laughing.  He's retired Army advanced radar and communications technician.  The F-22 has vector thrust but it makes a lot of noise and it still looses speed in hard maneuvers.  One thing this was not was winged aircraft, even an F-22 with vectored thrust.  It made maneuvers that only a helicopter could make but at a speed that had to be just below mach one at 7000 to 10000 ft. elevations.  The two sincere giveaways that this was a special craft was first it flew over an municipal airport and Air Force Base with no lights and in two flight paths of other aircraft and came very close to the rear of those two plane with no clearance lights or landing lightsâ€¦ all very illegal for either military or civilian aircraft.  The second giveaway was either drastic changes in direction or elevation did not change the velocity of the craft in the slightest.  I am not a believer in extraterrestrials, an engineer and would have to see either an extraterrestrial or their craft closely before I could possibly believe.  Someone owns and operates on incredible aircraft for sure.;38.8753097510373;-104.772105082987
8 ufo in the sky - looked like balloons but no baskets-stationary for over an hour now-teardrop when you zoom in - sitting over the cen;2021-02-01 09:00:00;1 hour ongoing;8 ufo in the sky - looked like balloons but no baskets-stationary for over an hour now-teardrop when you zoom in - sitting over the central part of the city for well over an hour...;38.8753097510373;-104.772105082987
