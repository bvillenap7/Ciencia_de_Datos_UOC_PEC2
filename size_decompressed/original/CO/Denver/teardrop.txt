I looked out and noticed waht appeared to be 2 Teardrop shaped balloons linked by a tether to one another, Metallic, shiny. The motion ;1998-05-26 10:40:00;15 seconds;I looked out and noticed waht appeared to be 2 Teardrop shaped balloons linked by a tether to one another, Metallic, shiny. The motion made when in flight was tandem. Could have been Weather balloons except for the fact they where moving against the prevailing winds 25/30 mph winds that day I bet. Lasted only a few seconds untill they moved behind cloud cover. Same as brief summary.. there wasn't much to it other than the fact they where not aircraft. they looked like this (>-----<) and moved in tandem ( like when you would spin a dumbell) one side is always opposite the other. The main action that made me believe this thing was unusual was A. It's appearance B. it's movement against the Wind.;39.7388117431815;-104.985074484367
Object had a tail and maintained shape and speed.;2016-01-10 16:53:00;5-10 seconds;Object had a tail and maintained shape and speed. One plume in west sky, first white then red closer to the horizon. Moved at a steady pace east to west then Disappeared below horizon. No sound, same shape for duration of visibility.;39.7388117431815;-104.985074484367
Falling star became closer to the ground by the seconds and appeared in the shape of an upside down teardrop floating above my car.;2017-10-17 01:00:00;1 minute;Falling star became closer to the ground by the seconds and appeared in the shape of an upside down teardrop floating above my car. Driving down 285 west of Santa Fe, there was what appeared to be a bright planet or star in the sky off in the West.  Driving farther west, object began dropping slowly, (almost like a falling star) closer to the ground becoming brighter.   When crossing over Federal, I drove right under the object, it had a bright light and was flashing red on the sides. It was in the shape of an upside down teardrop with circles around the top.  Object disappeared after driving under it.   After the experience, I felt extremely uneasy and eyesight was affected right after.  Almost like I came across a very powerful energy.;39.7388117431815;-104.985074484367
