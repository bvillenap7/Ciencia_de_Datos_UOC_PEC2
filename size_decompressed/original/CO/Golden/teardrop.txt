The metallic three pronged upside down teardrop object moved at a constant altitude and velocity Northwest in Golden, Colorado.;2008-05-03 20:25:00;30-40 seconds;The metallic three pronged upside down teardrop object moved at a constant altitude and velocity Northwest in Golden, Colorado. The metallic colored object with the shape of an upside down teardrop with three prongs moved Northwest past the baseball fields at Ulysses Park in Golden, Colorado at approximately 8:25 pm mountain standard time. The object appeared to be moving at the same altitude for the duration of the sighting. The object was roughly 100 feet away and likely 3-5 feet long across the side facing my friend and me. It disappeared from sight after moving at a constant rate of speed for about 30-40 seconds. The object made absolutely no sound. A group of skateboarders sitting about 25 feet away pointed out the object to my friend and me while we played catch as we did not notice it initially. Interestingly, one of the skateboarders made the comment, it's spying on you. My friend and I continued to play catch as the whole event felt somewhat surreal and there wasn't an instantaneous cohesive response by us. However, a few minutes later it grew too dark and we stopped playing catch. At that time we discussed the event and did not know what to make of it. As we walked down to the parking lot below the batting cages we hoped to see the skateboarders to ask them what they honestly thought it was but they were gone. After discussing the event, we postulated the only logical explanation was some sort of runaway kite but the seemingly constant altitude and velocity on a clear and breezeless night made that postulation unlikely to us. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));39.7598265306123;-105.220002040816
